index.html

```diff
@@ -190,5 +190,40 @@ <h2 class="aboutmeintro">About me</h2>
       </div>
 
     </section>
+    <footer>
+      <div class="footer">
+      <form class="form"action="https://formspree.io/f/mjvzwzlb" method="post">
+        <h2 class="footer-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <ul class="ul-form">
+          <li><input maxlength="30"
+            type="name"
+            placeholder="Full name" class="form-name"
+            required></li>
```

Comment:
Kindly note that your name is not being sent when the form is submitted, please fix this 🤗

![image](https://user-images.githubusercontent.com/73714615/192560006-129428e3-a945-4d4c-b084-90050adabf58.png)